tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,244 @@
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    }
+                  ]
+                }
+              },
+              "required": [
+                "granuleId",
+                "dataType",
+                "version",
+                "files"
+              ]
+            }
+          ]
+        }
+      },
+      "required": [
+        "granules"
+      ]
+    },
+    "config": {
```

Comment:
Does this pass tests with CMA? In my experience, "config" is a separate schema file (yeah, it's weird).

Reply:
On that note, you might need to remove the "input" parent object in this file.

Reply:
removed input. 

I did not get any unit test error.

Reply:
That doesn't surprise me; there weren't any schemas here before, so tests probably won't cover them. Implement the test recommended in the unit_test class, and that will help check your input/output schemas.

Reply:
added steps to validate schemas using `fastjsonchema` in `test_task_input_schema_return_error()`

Reply:
You need to run it through CMS via RunCumulusTask to get a realistic test. There's a lot of internal translation and modification in that class.

Reply:
added that test to run through CMS
`https://github.com/nasa/cumulus-orca/pull/126/commits/e3ffca84e241ae1c60e06c3408726980a8aa39f0`